.github/workflows/linux_meson.yml

```diff
@@ -121,6 +121,11 @@ jobs:
         export SCIPY_USE_PROPACK=1
         python dev.py --no-build test -j2 -- --durations 10 --timeout=60
 
+    - name: Run doctests
+      run: |
+        pip install git+https://github.com/ev-br/scpdt.git@1038618b83c9afa1fb79ca2f7839872708b7cae2
```

Comment:
why a specific hash?

Reply:
I wanted to run doctests with the most recent branch of scpdt, especially the one that deduplicates test items and exempts the tests directory.

Reply:
OK to keep for the time being; we'll need to transfer the whole package off my anyway. Will need to change it here when https://github.com/ev-br/scpdt/pull/91/ lands.